src/lib.rs

```diff
@@ -21,28 +21,26 @@
 //! project:
 //!
 //! ```
-//! extern crate airbrake;
-//!
-//! use std::num::ParseIntError;
-//!
-//! fn double_number(number_str: &str) -> Result<i32, ParseIntError> {
+//! fn double_number(number_str: &str) -> Result<i32, std::num::ParseIntError> {
```

Comment:
unnecessary change. iirc this might've been motivated by clippy lints, but I'll have to double check